### PR TITLE
[API-15237] - CCG auth added to the apply page

### PR DESCRIPTION
**Pull request by gileswells**
### Description

https://vajira.max.gov/browse/API-15237

This add some helper functions to the query.ts file and adds support to send CCG authorization params to LPB. This is done by changing all posted APIs that are requested from `{apiName}` to `{authType}/{apiName}`.

### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

File: src/containers/consumerOnboarding/components/sandbox/SandboxAccessSuccess.tsx [hunk 1]
```diff
@@ -83,7 +89,33 @@ const OAuthCredentialsNotice: React.FunctionComponent<OAuthCredentialsNoticeProp
...
+  selectedApis,
+}: OAuthCCGCredentialsNoticeProps) => {
+  const apiNameList = selectedApis.map(k => apisToEnglishOAuthList[k]);
+  const apiListSnippet = sentenceJoin(apiNameList);
+
+  return (
+    <div>
+      <p className="usa-font-lead">
+        <strong>Your VA API OAuth Client ID: </strong>
+        <span className="oauth-client-id">{ccgClientId}</span>
```
Comment (SlakrHakr) [inline src/containers/consumerOnboarding/components/sandbox/SandboxAccessSuccess.tsx:111]: im not seeing this if i sign up for only one ccg oauth api
image

Comment (SlakrHakr) [reply src/containers/consumerOnboarding/components/sandbox/SandboxAccessSuccess.tsx:111]: im always getting the above success page it seems like, regardless of which group of apis i select. shouldnt it be showing the client id/secret and apikey values here?

**Review by SlakrHakr — CHANGES_REQUESTED** [review]
the submission all looks and feels good
the success page doesnt seem to have all the information it typically has though

File: src/containers/consumerOnboarding/components/sandbox/SandboxAccessSuccess.tsx (hunk 1 above)
Comment (gileswells) [reply src/containers/consumerOnboarding/components/sandbox/SandboxAccessSuccess.tsx:111]: Well, I fixed it.

However if they pick one from each, I don't like it...
image

Comment (SlakrHakr) [reply src/containers/consumerOnboarding/components/sandbox/SandboxAccessSuccess.tsx:111]: yeah... its quite a lot... but all the info as there at least

**Review by SlakrHakr — APPROVED** [review]

**Pull request merged**